service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomServer.scala

```diff
@@ -22,11 +22,16 @@ import scala.language.experimental.macros
  */
 sealed trait LagomServer {
   val name: String
+  // TODO: replace with LagomServiceBinding[_] when removing LagomServer.forServices
   val serviceBindings: immutable.Seq[LagomServiceBinding[_]]
   def router: Router
 }
 
 object LagomServer {
+  def forService(binding: LagomServiceBinding[_]): LagomServer = {
+    forServices(binding)
+  }
```

Comment:
I'm wondering if we should create an even simpler API, like `LagomServer[ServiceInterface](wire[ServiceImpl])`.